docs/src/main/paradox/release-notes/2.0.x.md

```diff
@@ -15,9 +15,106 @@ To understand the forces on version numbers, read about @ref:[Alpakka's versioni
 In case you are browsing a specific version's documentation: check out the [latest release notes](https://doc.akka.io/docs/alpakka/current/release-notes/)
 @@@
 
-All Alpakka modules are available for Scala 2.13 and all Alpakka modules are compatible with Akka 2.6.1+ and Akka 2.5.27+. An exception is MQTT streaming which requires Akka 2.6.1.
+All Alpakka modules are available for Scala 2.13 and all Alpakka modules are compatible with Akka 2.6.4+ and Akka 2.5.31+. An exception is MQTT streaming which requires Akka 2.6.4.
```

Comment:
Does `An exception is MQTT streaming which requires Akka 2.6.4.` mean that `MQTT` requires _exactly `2.6.4`?

It looks like the whole paragraph originally had different versions but bulk replace changed the meaning?

I'm not sure I understand what the minimum/exact versions are.